pkgs/applications/networking/mailreaders/thunderbird-bin/default.nix

```diff
@@ -112,21 +118,20 @@ stdenv.mkDerivation {
       ln -s "$prefix/usr/lib/thunderbird-bin-${version}/thunderbird" "$out/bin/"
 
       for executable in \
-        thunderbird mozilla-xremote-client thunderbird-bin plugin-container \
-        updater
+        thunderbird crashreporter thunderbird-bin plugin-container updater
       do
         patchelf --interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
           "$out/usr/lib/thunderbird-bin-${version}/$executable"
       done
 
       for executable in \
-        thunderbird mozilla-xremote-client thunderbird-bin plugin-container \
-        updater \
+        thunderbird crashreporter thunderbird-bin plugin-container updater \
         components/libdbusservice.so components/libmozgnome.so \
         libnssdbm3.so libsmime3.so libxul.so libprldap60.so libnss3.so \
         libplc4.so libfreebl3.so libmozsqlite3.so libmozalloc.so libnspr4.so \
         libssl3.so libldif60.so libsoftokn3.so libldap60.so libnssutil3.so \
-        libnssckbi.so libplds4.so
+        libnssckbi.so libplds4.so \
+        'distribution/extensions/{e2fda1a4-762b-4020-b5ad-a41df1933103}/components/libcalbasecomps.so'
```

Comment:
Maybe this can be changed to something like `$(find -name "*.so")` so that you don't have to enumerate all .so's.